src/Redux/__Tests__/Main.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import renderer from 'react-test-renderer';
+import Main from '../pages/Main';
+import configureStore from '../Redux/ConfigureStore';
+
+test('Main renders correctly', () => {
+  const tree = renderer
+    .create(
+      <Provider store={configureStore()}>
+        <Main />
+      </Provider>
+    )
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] your tests are still failing kindly fix
![image](https://user-images.githubusercontent.com/66097371/206552183-fd5067fe-f962-45a0-92b4-d61a61cb0eb1.png)

Reply:
@VICTOR-CHUKWU if you have some time could you help me with these tests? i don't know why they keep failing

Reply:
or @GzimAsani if you have some time could you help me with these tests....

Reply:
so now it is working I had a problem with my local I had to use gitpod.io to test but now they are working 

https://david63011-reactcapston-n74sjpnn2e0.ws-us78.gitpod.io/

Reply:
Tests are now passing 

![image](https://user-images.githubusercontent.com/86318284/206745139-2866767c-14dc-4460-9c27-9a3b0533d417.png)